### DISCUSSION TITLE
Switch between multiple NocoDB accounts in NocoDB?

### DISCUSSION THREAD

**Discussion by ghost** [nocodb/nocodb#7290] [Q&A] [answered]
Hey y'all!

***Problem*** 
There is currently no feature to switch between accounts in NocoDB. However, there are several solutions(Gmail, Yahoo etc) that allow you to switch between accounts.

***What would this feature look like?***
For example, Gmail (client email) has the following feature: [Switch between multiple Google accounts](https://support.google.com/docs/answer/2405894?hl=en&co=GENIE.Platform%3DAndroid), see this: 
> *"[...] If you have more than one Google Account, you can sign in to multiple accounts at once. That way, you can switch between accounts without signing out and back in again. [...]"*

And here:
> *"[...] All you need to do is swipe upward on your profile icon at the top-right corner to switch to the next account in the list . You can also swipe downward to access the last account in the list, or simply move back to the previous account after switching." [...]*

This would be similar to what is being described here too. In theory if the user has a NocoDB account in an organizational sense, where there is sharing certain information with the members. It would make sense to have that kind of feature. The initial advantage of this is that you would not need to login again with the email, but just enter the password.

***ui/ux***
The image below is an illustration of the Gmail feature that is account switching, but this same illustration illustrates my concept of switching NocoDB accounts.
<img src="https://github.com/nocodb/nocodb/assets/154566915/bb88af9b-8be7-473e-a452-126db0b0dc6b" width='760px'/>

**Image rights belong to Lifewire** `https://www.lifewire.com/how-to-switch-fast-between-multiple-gmail-accounts-1171971` 

***disclaimer***
1. I do not work for NocoDB and have no connection with the company. In my case, I’m a NocoDB user and would like to contribute with some ideas.
2. These designs are not intended to endorse any position of the company or mine, they are just visual demonstrations about the feedbacks I read here.
3. I would like to contribute to this technical discussion through screen prototypes.
4. I hope to make an interesting view for most people.
5. I’m not promoting Gmail, Google, Yahoo, LifeWire ... just referring in bibliographic terms to something that could be developed or not. 
6. I would like to contribute to this resource, but I don't know how to start, I know little about programming.
7. This is more of a question than a feature request.
8. I don't have any money, but I can help implement this feature if it's interesting for everyone here.

***faq***
*Have you all thought about creating the feature to switch between multiple NocoDB accounts in NocoDB?*

**Comment by dstala** [comment, marked as answer]
In NocoDB cloud hosted solution (currently in free technical preview), we do have concept for Workspaces. That should solve the problem under consideration to a good extent I believe. 

https://docs.nocodb.com/workspaces/workspace-overview

**Reply by ghost** [reply]
Hi @dstala, how are you? thanks for comment!
